README.md

```diff
@@ -372,12 +417,24 @@ callback let you add assertions and enrich the test.
     Check if a log message sent by the daemon has been correctly received. A log
     message is caracterized by its *api* and the message level *error*,
     *warning*, *notice*, *info* or *debug*.
+* **_AFT.assertEvtNotReceived(eventName)**
```

Comment:
Add a timeout event as last arguments even if not implemented right now, it shouldn't be long. So for one time we have the documentation more advanced than the code...